### DISCUSSION TITLE
HELP. This has appeared and I don't know what to do

### DISCUSSION THREAD

**Discussion by ChocokrispisToto** [Monika-After-Story/MonikaModDev#9869] [QA - tech support + general]
![Captura](https://user-images.githubusercontent.com/122856596/213840880-a0568a7e-4a48-458d-abba-a32bb2bd723b.PNG)

This appeared to me while I was playing "pong", and no matter how much I hit "ignore" I can't continue with the game.
Does anyone know how to fix this code?

**Comment by TooMuchInvigilation** [comment]
Hi. It looks like you are missing a file called "acs-mug-0.png". I would suggest checking if it is present in the "game/mod_assets/monika/a/" folder.
![image](https://user-images.githubusercontent.com/122057575/213874868-db5578b9-6cde-4e19-858d-17b8ad47c36b.png)
